rules/declaration-strict-value.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+// See https://github.com/AndyOGo/stylelint-declaration-strict-value
+
+const properties = [
+    '/color/',
+    'font-family',
+    'font-weight',
+    'z-index',
```

Comment:
I think that  we also can add  `fill`and `stroke` to the properties that is used to color `svg`

Reply:
Added `fill` and `stroke` properties.